docs/source/loggers/tensorboard_logger.md

```diff
@@ -0,0 +1,198 @@
+from composer.loggers import TensorboardLogger
+
+transform = transforms.Compose([transforms.ToTensor()])
+train_set = datasets.MNIST("data", train=True, download=True, transform=transform)
+val_set = datasets.MNIST("data", train=False, download=True, transform=transform)
+train_dataloader = DataLoader(train_set, batch_size=128)
+eval_dataloader = DataLoader(val_set, batch_size=64)
+
+# Create your Tensorboard Logger here.
+tb_logger = TensorboardLogger(log_dir="./my_tensorboard_logs")
+
+trainer = Trainer(
+    model=mnist_model(num_classes=10),
+    train_dataloader=train_dataloader,
+    eval_dataloader=eval_dataloader,
+    max_duration='5ep',
+    # Add your Tensorboard Logger to the trainer here.
+    loggers=[tb_logger],
+    eval_interval='1ep'
+)
+trainer.fit()
+```
+
+Now, run this code and if all goes well, your loss and metric results will be logged Tensorboard log files, which will be written to "./my_tensorboard_logs". 
+See [these instructions](#viewing-your-results-locally) for viewing your results in the Tensorboard viewer.
+
+## Logging to Tensorboard using YAML configs
+To log to Tensorboard using YAML config files, you can add these lines to you YAML file:
+```yaml
+  loggers:
+    tensorboard:
+      log_dir: my_tensorboard_logs    
+```
+Putting it all together the MNIST config YAML file version of the code above should look like:
+
+```yaml
+train_dataset:
+  mnist:
+    is_train: true
+    datadir: ./datasets/mnist
+    download: true
+val_dataset:
+  mnist:
+    is_train: false
+    datadir: ./datasets/mnist
+    download: true
+model:
+  mnist_classifier:
+    num_classes: 10
+max_duration: 5ep
+eval_interval: 1ep
+train_batch_size: 128
+eval_batch_size: 64
+loggers:
+  tensorboard:
+    log_dir: my_tensorboard_logs    
+```
+Then save that YAML file as `mnist.yaml` for example.
+
+To run the training using this config file, just run:
+
+```bash
+python examples/run_composer_trainer.py -f mnist.yaml
+```
+
+If all goes well, your loss and metric results will be logged Tensorboard log files, which will be written to "./my_tensorboard_logs". 
+See [these instructions](#viewing-your-results-locally) for viewing your results in the Tensorboard viewer.
+
+## Saving your Logs to S3
+If you run your training on a cluster without persistent local storage or you don't have ssh
+access to your cluster, you may not be able to access your Tensorboard logs after the training
+run is over. To solve this issue you can have your Tensorboard logs saved to some cloud storage
+somewhere, like Amazon S3. 
+
+<!-- I can't figure out how to use the ObjectStoreLogger pythonically, so punting on this -->
+<!-- To save your logs to S3, you will need create an  `ObjectStoreLogger` object, like so:
+
+```python
+from composer.loggers import ObjectStoreLogger
+from composer.utils import
+``` -->
+
+If you are using YAML configs you can get S3 logging by adding the following lines to your YAML file under loggers:
```

Comment:
Should probably link to the `ObjectStoreLogger` docs

Reply:
done